### PR TITLE
[[Bug 20645]] Create mobile scroller each time DataGrid is opened

### PR DESCRIPTION
**Pull request by trevordevore**

### CODE REVIEW

**Comment by trevordevore** [comment]
Note to reviewer: I haven't tested this yet as I only have XCode 9.1 installed on my computers and LC doesn't support Xcode 9.1 yet. Bug report can be found here:

http://quality.livecode.com/show_bug.cgi?id=20645

File: Toolset/palettes/revdatagridlibrary/behaviorsdatagridbuttonbehavior.livecodescript [hunk 1]
```diff
@@ -210,12 +195,34 @@ private command _Initialize
       unlock messages
       
       put true into sInit
+   else
```
Comment (montegoulding) [inline Toolset/palettes/revdatagridlibrary/behaviorsdatagridbuttonbehavior.livecodescript]: @trevordevore thanks for this. It looks good the only issue is I think this needs to be guarded against execution on non-mobile platforms. So `else if the environment is "mobile" then`

Comment (trevordevore) [reply Toolset/palettes/revdatagridlibrary/behaviorsdatagridbuttonbehavior.livecodescript]: Fixed.

**Comment by montegoulding** [comment]
@livecode-vulcan review ok a6ddd9a

Thanks @trevordevore

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok a6ddd9ac230e8dcb4abd81d9de39efaa0da88bc5

**Pull request merged**